### PR TITLE
Refine en doc

### PR DESCRIPTION
**Pull request by JiabinYang**
MacHi 2019-10-10 17-09-11

### CODE REVIEW

File: paddle/fluid/pybind/imperative.cc [hunk 1]
```diff
@@ -180,28 +180,36 @@ void BindImperative(py::module *m_ptr) {
   py::class_<imperative::detail::BackwardStrategy> backward_strategy(
       m, "BackwardStrategy", R"DOC(
 
-    BackwardStrategy is a descriptor of a how to run the backward process. Now it has:
+    BackwardStrategy is a descriptor of a how to run the backward process.
```
Comment (sneaxiy) [inline paddle/fluid/pybind/imperative.cc]: `a how` -> `how`?

File: paddle/fluid/pybind/imperative.cc [hunk 2]
```diff
@@ -180,28 +180,36 @@ void BindImperative(py::module *m_ptr) {
...
-    1. :code:`sort_sum_gradient`, which will sum the gradient by the reverse order of trace.
+    **Note**:
+        **This API is only avaliable in** `Dygraph <../../user_guides/howto/dygraph/DyGraph.html>`_ **Mode**
 
-    Examples:
+    Attribute:
+        **sort_sum_gradient**:
 
-        .. code-block:: python
+        If it will sum the gradient by the reverse order of trace. eg. x_var ( :ref:`api_guide_Variable` ) will be the input of multiple OP such as :ref:`api_fluid_layers_scale` , this attr will decide if it will sum gradient of `x_var` by the reverse order.
```
Comment (sneaxiy) [inline paddle/fluid/pybind/imperative.cc]: If it is true, framework will sum...

File: paddle/fluid/pybind/imperative.cc (hunk 1 above)
Comment (JiabinYang) [reply paddle/fluid/pybind/imperative.cc]: done

File: paddle/fluid/pybind/imperative.cc (hunk 2 above)
Comment (JiabinYang) [reply paddle/fluid/pybind/imperative.cc]: done

**Review by wanghaoshuang — DISMISSED** [review]

**Review by sneaxiy — APPROVED** [review]

**Review by guoshengCS — APPROVED** [review]

**Review by Superjomn — APPROVED** [review]

**Pull request merged**